web/router.ex

```diff
@@ -11,9 +11,7 @@ defmodule RemoteRetro.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    if Application.get_env(:remote_retro, :add_mock_user_to_session) do
-      plug :add_mock_user_to_session
-    end
+  
```

Comment:
and i don't want a network connection + ngrok to be necessary for devs to work on the project